go.mod

```diff
@@ -74,7 +74,6 @@ require (
 	github.com/go-logr/logr v1.2.4
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.11.0
-	github.com/go-redis/redis/v9 v9.0.0-rc.1 // replaced
```

Comment:
Please update .github/dependabot.yml and .github/renovate.json too.

Reply:
does this look ok?
[835c40b](https://github.com/gravitational/teleport/pull/30294/commits/835c40b1faa6e884e914dde80ac48799731a4c1d)

Reply:
Tes, thanks!